DataManagement/GitTutorial.Rmd

```diff
@@ -297,11 +357,83 @@ data/
 <br/><br/>
 <br/><br/>
 
+
```

Comment:
having had this problem today, I think it needs to be data/*, otherwise the exception does not work